packages/cms-cli/commands/init.js

```diff
@@ -31,11 +31,13 @@ const { logDebugInfo } = require('../lib/debugInfo');
 const COMMAND_NAME = 'init';
 
 const oauthConfigSetup = async ({ configPath }) => {
+  trackCommandUsage(COMMAND_NAME, {
```

Comment:
I think we will need to use a different event for tracking progress so that we don't inflate command usage and get an accurate breakdown of the commands.